src/components/NoticeOfWithdrawal/RecordToBeWithdrawn.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <VcardTemplate id="record-to-be-withdrawn">
+    <template #icon>
+      mdi-file-document-outline
+    </template>
+
+    <template #title>
+      Record to be Withdrawn
+    </template>
+
+    <template #content>
+      <v-row
+        no-gutters
+        class="px-6 py-7"
+      >
+        <v-col
+          cols="12"
+          sm="3"
+          class="pr-4"
+        >
+          <strong>Filing Type</strong>
+        </v-col>
+        <v-col
+          id="withdrawal-filing-type"
+          cols="12"
+          sm="9"
+        >
+          {{ getFilingToBeWithdrawn().displayName }}
+        </v-col>
+      </v-row>
+
+      <v-row
+        no-gutters
+        class="px-6 py-7"
+      >
+        <v-col
+          cols="12"
+          sm="3"
+        >
+          <strong>Filing Number</strong>
+        </v-col>
+        <v-col
+          id="withdrawal-filing-number"
+          cols="12"
+          sm="9"
+        >
+          {{ filingToBeWithdrawn }}
+        </v-col>
+      </v-row>
+
+      <v-row
+        no-gutters
+        class="px-6 py-7"
+      >
+        <v-col
+          cols="12"
+          sm="3"
+        >
+          <strong>Effective Date and Time</strong>
+        </v-col>
+        <v-col
+          id="withdrawal-filing-date"
+          cols="12"
+          sm="9"
+        >
+          {{ getFormattedFilingDate() }}
+        </v-col>
+      </v-row>
+    </template>
+  </VcardTemplate>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator'
+import { Getter } from 'pinia-class'
+import { VcardTemplate } from '@/components/common'
+import { ApiFilingIF } from '@/interfaces'
+import { useFilingHistoryListStore } from '@/stores/filingHistoryListStore'
+
+  @Component({
+    components: {
+      VcardTemplate
+    }
+  })
+export default class RecordToBeWithdrawn extends Vue {
+    @Prop({ required: true }) readonly filingToBeWithdrawn!: string
+
+    @Getter(useFilingHistoryListStore) getFilings!: Array<ApiFilingIF>
+
+    getFilingToBeWithdrawn (): ApiFilingIF| undefined {
+      const filingId = Number(this.filingToBeWithdrawn)
+      const filings = this.getFilings as ApiFilingIF[]
+      return filings.find(filing => filing.filingId === filingId)
+    }
+
+    getFormattedFilingDate (): string {
+      const filing = this.getFilingToBeWithdrawn()
+      if (filing && filing.effectiveDate) {
+        const date = new Date(filing.effectiveDate)
```

Comment:
Use Date Mixin to convert between date formats.

Reply:
Using Date Mixin, the "am" is not capitalized, while in design it's "AM":
![image](https://github.com/user-attachments/assets/d38ceb47-ea35-4c7d-a64e-c435e47d36aa)

Is it ok?

Reply:
Use the mixin so that the display is consistent throughout the app.

Let Jacqueline know. If the new design is to capitalize it then we will need a new ticket so we can make the change to the  mixin in all UIs.

Reply:
I think you shouldn't use `new Date()` on line 100 either. There's a Date Mixin function to do this.

The reason for the Date Mixin is to handle absolute timezones (esp if the user is not in Pacific tz, and also because we can't trust the user's browser settings).

Reply:
The filing.effectiveDate is in "Wed, 27 Nov 2024 17:34:07 GMT" format. I didn't find a Date Mixin function to handle this format. Do you have suggestions? Thanks a lot

Reply:
![image](https://github.com/user-attachments/assets/f6278adf-2e00-492b-90bd-8ca676a55f49)

Then you are right -- using `new Date()` is acceptable here. Resolving!